sdks/how-to/how-to-manage-messages.md

```diff
@@ -0,0 +1,150 @@
+---
+slug: how-to-manage-messages
+description: Learn how to manage Messages
+tags:
+  - Message
+  - Chat
+  - Messaging
+  - Channel
+---
+{{#ehrlite}}
+# Handling messages
+
+In this section, we will learn how to manage Messages. A `Message` is a text exchange between `activeParticipants` of a `Topic`.
+
+## Creating a Message
+
+To create a `Message`, you must use the `create` function. To do this, you will need to provide the following information:
+
+- `topic`, a reference to the `Topic` to which you wish to add a message.
+- `content`, the content of the `Message`. (optional)
+- `attachments`, the attachments of the `Message`. (optional)
+- `tags`, the `Tags` related to the `Message`. (optional)
+- `codes`, the `Codes` related to the `Message`. (optional)
```

Comment:
I think this may be confusing for people who don't know what codes are. It may be worth to specify what they are and in what they differ from tags.